deploy/crds/interconnectedcloud_v1alpha1_interconnect_crd.yaml

```diff
@@ -69,6 +69,9 @@ spec:
                 livenessPort:
                   type: integer
                   description: The liveness probe port number
+            users:
```

Comment:
Should this be `user` as plural would indicate array of users?

Reply:
There can be more than one user in the secret (if the operator genrates it there is only one, but the secret can be provided by the creator of the CR also).